addons/crm/models/res_partner.py

```diff
@@ -64,6 +64,11 @@ def action_view_opportunity(self):
         '''
         action = self.env['ir.actions.act_window']._for_xml_id('crm.crm_lead_opportunities')
         action['context'] = {}
+
+        list_view = [(state, view) for state, view in action['views'] if view == 'list']
+        other_views = [(state, view) for state, view in action['views'] if view != 'list']
```

Comment:
Could we have a small test case? (If possible)

Reply:
Hello!
I have added a test case. Please let me know if it seems trivial.
Thanks :smile: